src/passwordPenalty.js

```diff
@@ -1,23 +1,13 @@
-/**
- * Calculates the sum of penalty points for a given password.
- * Double characters like `aa` count as 1 penalty point, triples and more are 2 points.
- * It returns the sum of penalty points for a password or 0.
- * @param {string} password
- * @returns {number}
- */
-export default function penaltyPoints(password = "") {
-  if(!password) return 0;
+export default function penaltyPoints(password) {
+  if (!password) return 0;
 
-  const SequenceNumber = password?.match(/(.)\1+/gm)
-  
-    let penalities = 0;
+  let penalities = 0;
 
-    SequenceNumber?.forEach(element => {
-      if(element.length > 2){
-        penalities +=2;
-      }else {
-        penalities++;
-      }
-    });
-    return penalities;
+  // The following line contain a regex that look for any character that repeat it self as a group.
+  password.match(/(.)\1+/gm)
+    ?.forEach((element) =>
+      element.length > 2 ? (penalities += 2) : penalities++
+    );
+
+  return penalities;
```

Comment:
I like how you have simplified this, However I feel you can still simplify it more as shown below by removing the last return.
```suggestion
  return password.match(/(.)\1+/gm)
    ?.forEach((element) =>
      element.length > 2 ? (penalities += 2) : penalities++
    );
```